extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -164,8 +164,12 @@ export abstract class DacFxConfigPage extends BasePage {
 	}
 
 	protected getRootPath(): string {
-		// return rootpath of opened folder in file explorer if one is open, otherwise default to user home directory
-		return vscode.workspace.rootPath ? vscode.workspace.rootPath : os.homedir();
+		// use previous file location if there was one
+		if (this.fileTextBox.value && path.dirname(this.fileTextBox.value)) {
```

Comment:
do we need to check this directory's existence in case its deleted or something?

Reply:
I think it's ok to let DacFx handle if the directory doesn't exist. The extract fails with this error:
![image](https://user-images.githubusercontent.com/31145923/62581751-f21e0100-b85e-11e9-9a78-9d7f8d8b6f20.png)